scripts/packages/chocolatey/bazel.nuspec.template

```diff
@@ -77,7 +77,7 @@ Supply like `--params="/option:value ..."` ([see docs for --params](https://gith
     <dependencies>
       <dependency id="jdk8" version="[8.0.102,)"/>
       <dependency id="msys2" version="[20160719.1,)"/>
-      <dependency id="python2" version="[2.7.11,3.0)"/>
+      <dependency id="python" version="[2.7.11,)"/>
```

Comment:
@petemounce this is effectively the same as `version="2.7.11"` - it just means 2.7.11 is the minimum version that can be installed.